swarm/network/stream/peer.go

```diff
@@ -166,7 +166,7 @@ func (p *Peer) SendOfferedHashes(s *server, f, t uint64) error {
 		"send.offered.hashes")
 	defer sp.Finish()
 
-	hashes, from, to, proof, err := s.SetNextBatch(f, t)
+	hashes, from, to, proof, err := s.setNextBatch(f, t)
```

Comment:
Why do we return `proof` in `setNextBatch` and in `SetNextBatch` ? This is always `nil`. This will simplify the code further down as well.

Let's not include prematurely code that will be implemented in the future. When we implement those proofs, then we change the interface of these methods. Right now we are just increasing the complexity of the code, introducing statements full of no-ops.

Reply:
agreed, this is leftover code from when we had the light node streams.
But do you want me to put this change in this PR?